### PR TITLE
Add activity for `bid` events

### LINKED ISSUE #9: Save Activity details for bids
## Changes

- Activities are not saving in the subgraph for bid related things.

### LINKED ISSUE #11: Fix bid handler
image

### PR DESCRIPTION
**Pull request by AshenI99**
- Changed the Activity entity in subgraph for a better structure. Took the inspiration from the foundation graph. (https://thegraph.com/hosted-service/subgraph/f8n/fnd)

**Previous Structure**

Screenshot 2022-03-03 at 18 42 32

**Updated Structure**

Screenshot 2022-03-03 at 18 42 14

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -90,15 +90,9 @@ type Activity @entity {
...
-  # mint activity
-  mintBy: User
-
-  # burn activity
-  burnBy: User
-
-  # transfer activity
-  transferFrom: User
-  transferTo: User
+  actor: User
```
Comment (TheLubab) [inline schema.graphql]: I just suggest calling this `sender` to keep it consistent with near contract `context.sender`

**Review by TheLubab — APPROVED** [review]
This structure is a bit confusing to understand, specailly if you see it for the first time. But if we are going to use it only in the History component, then that's okay

File: schema.graphql (hunk 1 above)
Comment (AshenI99) [reply schema.graphql:93]: @achrafismyname I think, we should add descriptions for the entities. Then It will be easier to understand.

**Comment by AshenI99** [comment]
@achrafismyname I think, we should add descriptions for the entities. Then It will be easier to understand.

**Pull request merged**